scripts/symcheck.py

```
@@ -0,0 +1,322 @@
+            print(f"  [缺类 missing type]   {t}")
+        for m in missing_members:
+            print(f"  [缺成员 missing member] {m}")
+        if not missing_types and not missing_members:
+            print("  （无 / none）")
+        print()
+
+        total = len(missing_types) + len(missing_members)
+        print(f"缺失合计 / total missing: {total}")
+        if total:
+            print()
+            print("两类成因，修法相反 —— 看缺失符号里写的是哪一代类型：")
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)